Classes/ScrnChainsaw.uc

```diff
@@ -177,7 +195,8 @@ simulated function ClientReload()
 simulated function ActuallyFinishReloading()
 {
     super.ActuallyFinishReloading();
-
+    if (FakedGasCan != none)
+    FakedGasCan.SetDrawScale(0.01); //test
```

Comment:
Attached actors with modified relative location or rotation do significant performance impact while drawing. I'm not sure if that has any effect on modern computers, but for toaster-players set `FakedGasCan.bHidden` to avoid drawing and potential framerate drop.

Reply:
I don't know how much performance impact it has, but I tested the Neostead 2000 (It's in LaS Weapon Pack) in solo with uncapped frames. The Neostead 2000 has 8 attached fakedshells each with individually specified location and rotation. I get 400fps looking in a corner, but when I switch to M4 M203 it changes to 380fps. It's not a very indicative test but I think the performance impact still isn't significant.
I can add a var for toaster players just in case though, but where should it be put?